.github/CODEOWNERS

```
@@ -47,6 +48,9 @@
 
 ## integration tests for auto-instrumentations
 /tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/                                                @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DataStreamsMonitoring*                          @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet  @DataDog/data-streams-monitoring
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/DataStreamsMonitoring*                      @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet  @DataDog/data-streams-monitoring
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Azure/DataStreamsMonitoring*                    @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet  @DataDog/data-streams-monitoring
```

I think we can do this to cover all existing and future directories? This way we don't have to keep adding new directories. (assuming this works in `CODEOWNERS` 😅 )
```suggestion
/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/**/DataStreamsMonitoring*                       @DataDog/apm-idm-dotnet  @DataDog/tracing-dotnet  @DataDog/data-streams-monitoring
```